doc/guides/contributing/pull-requests.md

```diff
@@ -192,9 +192,6 @@ One of the existing contributors will help get things situated and the
 contributor landing the Pull Request will ensure that everything follows
 the project guidelines.
 
-See [core-validate-commit](https://github.com/nodejs/core-validate-commit) -
-A utility that ensures commits follow the commit formatting guidelines.
-
```

Comment:
If we want to keep a reference to [core-validate-commit](https://github.com/nodejs/core-validate-commit) then we should instruct to run with `--no-validate-metadata` as we do in [Travis](https://travis-ci.com/nodejs/node/jobs/266712335#L258) for pull requests.
https://github.com/nodejs/node/blob/357a99293e0132ed96afde56df5ce5aea2b550cd/tools/lint-pr-commit-message.sh#L38

Reply:
Has anyone been able to run all of the CI checks locally? I know Travis is open source, so it doesn't seem like a far-fetched notion. If the goal is to prevent a single failing CI build from occurring (which would produce the error), running the whole gamut of checks locally would likely be more effective.

Reply:
Ideally, the `make test` target should include a commit tester... though I suspect there are some complications, like only linting the new commit messages, not every one in the history of node.js,and perhaps dealing with fixup commits, etc.

Reply:
@sam-github, are you saying that this check should take place when running the command mentioned in [Step 6: Test](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#step-6-test)? If so, simply validating the most recent commit should be all that is necessary as prior commits can probably be safely assumed to be valid having come from the upstream master. It's probably important to keep this discussion within the scope of ensuring a successful process for those following this guide.